core/domain/stats_services.py

```diff
@@ -47,14 +46,11 @@ def _migrate_to_latest_issue_schema(exp_issue_dict):
         Exception. The issue_schema_version is invalid.
     """
     issue_schema_version = exp_issue_dict['schema_version']
-    if issue_schema_version is None or issue_schema_version < 1:
-        issue_schema_version = 0
-
-    if not (0 <= issue_schema_version
+    if not (1 <= issue_schema_version
             <= stats_models.CURRENT_ISSUE_SCHEMA_VERSION):
         raise Exception(
             'Sorry, we can only process v1-v%d and unversioned issue schemas at'
-            'present.' %
+            ' present.' %
```

Comment:
Nice catch, please add spaces to line 52 though.

Reply:
Done